test/snippet/alphabet/quality/phred94.cpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/quality/phred94.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    seqan3::phred94 phred;
+    phred.assign_rank(2); // wrapper for assign_phred(2)
+    seqan3::debug_stream << phred.to_phred() << "\n"; // 2
+    seqan3::debug_stream << phred.to_char() << "\n";        // '#'
+    seqan3::debug_stream << phred.to_rank() << "\n";  // 2
```

Comment:
```suggestion
 seqan3::debug_stream << phred.to_char() << "\n"; // '#'
```

Reply:
Could you explain, why you would change from to_rank() to to_char()? I thougt we wanna show all possible functions for phred here.

Reply:
Ups, I copied the wrong line, I just wanted you to line up the comment. :sweat_smile: